velox/exec/HashBuild.cpp

```diff
@@ -1115,7 +1115,7 @@ void HashBuild::reclaim(
   for (auto* op : operators) {
     HashBuild* buildOp = static_cast<HashBuild*>(op);
     spillTasks.push_back(
-        std::make_shared<AsyncSource<SpillResult>>([buildOp]() {
+        memory::createAsyncMemoryReclaimTask<SpillResult>([buildOp]() {
```

Comment:
The VELOX_CHECK exception in createAsyncMemoryReclaimTask method return lambda is not handled. It might fail silently

Reply:
The two check failures in createAsyncMemoryReclaimTask will fail the query? Thanks!